messaginginapp/src/main/java/io/customer/messaginginapp/extension/GistExtensions.kt

```diff
@@ -0,0 +1,22 @@
+package io.customer.messaginginapp.extension
+
+import build.gist.data.model.Message
+import io.customer.messaginginapp.type.InAppMessage
+import io.customer.sdk.extensions.random
+
+fun Message.getNewFromInApp(
+    message: InAppMessage
+): Message = Message(
+    instanceId = message.instanceId,
+    messageId = message.messageId,
+    properties = mapOf(
+        Pair(
+            "gist",
+            mapOf(
+                Pair("campaignId", message.deliveryId)
+            )
+        )
+    )
+)
```

Comment:
Looks like we aren't using the receiver `Message`, so this should have been on `InAppMessage` instead?

Reply:
Sorry, I don't understand your question. 

Is the question you have a question that Shahroz asked in this PR or a different question?

Reply:
I think what Rehan meant to say is, is an extension method on `build.gist.data.model.Message` and we aren't using the receiver "build.gist.data.model.Message" but we are using the `InAppMessage` model only.

So would it make more sense to have "InAppMessage" as a receiver rather than "build.gist.data.model.Message" which isn't being used.

Reply:
Shahroz got this right. I was suggesting this method to be following instead:
```
fun InAppMessage.getNewMessage(): Message
```
As I don't see `this` receiver used anywhere in the extension.